src/applications/disability-benefits/all-claims/config/781/index.js

```diff
@@ -52,13 +52,6 @@ export function createFormConfig781(iterations) {
         uiSchema: incidentSupport.uiSchema('781'),
         schema: incidentSupport.schema,
       },
-      [`incidentDate${index}`]: {
-        title: `${numberToWords[index]} PTSD incident date`,
```

Comment:
Same with this one, put this title on the incidentDate page that is in the file.